src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -0,0 +1,410 @@
+            return HypothesisScore(0.5, 0.5, 0.5, 0.5, 0.5, 0.5)
+        
+        section = match.group(1)
+        
+        # Extract individual scores
+        def extract_score(criterion: str, text: str) -> float:
+            pattern = f'{criterion}:\\s*([0-9.]+)'
+            match = re.search(pattern, text, re.IGNORECASE)
+            if match:
+                try:
+                    return float(match.group(1))
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)